docs/get-started.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Get Started
+layout: default
+nav_order: 2
+---
+
+# Get Started
+Doc Detective is versatile, and you can deploy it in many ways to suit the requirements of your development environment. This guide covers the three most common deployment methods: NPM, CLI, and Docker.
```

Comment:
Remove mention of Docker. It's still going to happen, but it's not as high a priority as it once was.